### DISCUSSION TITLE
Javascript enhancement

### DISCUSSION THREAD

**Discussion by twilson9034** [Budibase/budibase#3797] [Ideas]
It would be nice if the entry field for JavaScript could have a popout feature to allow for a bigger viewing area. Also maybe the ability to have autocomplete when writing JavaScript in the entry field. Like on Playcode.io 
![image](https://user-images.githubusercontent.com/94488623/146249211-6997d9d6-e4f3-441c-ae5c-1fb67bdb4fa4.png)

**Comment by kshep92** [comment]
Code completion assistance in all fields where handlebars templates are accepted would be a nice addition as well.

**Reply by bjornclauw** [reply]
`CTRL+space` kinda does this for the javascript editor.
